src/hammer-vlsi/hammer_vlsi.py

```diff
@@ -512,6 +531,37 @@ def get_setting(self, key: str):
         except AttributeError:
             raise ValueError("Internal error: no database set by hammer-vlsi")
 
+    def create_enter_script(self, enter_script_location: str = "") -> None:
+        """
+        Create the enter script inside the rundir which can be used to
+        create an interactive environment with all the same variables
+        used to launch this tool.
+
+        :param enter_script_location: Location to create the enter script. Defaults to self.run_dir + "/enter"
+        """
+        if enter_script_location == "":
+            enter_script_location = os.path.join(self.run_dir, "enter")
+        enter_script = reduce(lambda a, b: a + "\n" + b, map(lambda k_v: "export {0}=\"{1}\"".format(k_v[0], k_v[1]), self.env_vars.items()))
```

Comment:
Isn't there some sort of join function on strings that could be used instead of reduce?